js/views/serverConnectModal.js

```diff
@@ -188,6 +190,18 @@ module.exports = baseModal.extend({
     }
   },
 
+  sslOn: function(){
+    this.model.set('SSL', false);
+    this.$('.js-sslOn').addClass('hide');
```

Comment:
for both this and sslOff, I think you could just re-render, rather than having to manually manipulate the dom.

alternatively, and maybe even better, you could bind a change:SSL handler in initialize and set that to re-render.